custom_components/pyscript/trigger.py

```diff
@@ -644,6 +644,112 @@ def start(self):
             self.task = Function.create_task(self.trigger_watch())
             _LOGGER.debug("trigger %s is active", self.name)
 
+    def ident_any_values_changed(self, func_args):
+        """Check for changes to state or attributes on ident any vars"""
+        value = func_args.get('value')
+        old_value = func_args.get('old_value')
```

Comment:
Should the prior two lines be below the next `if` statement?  I'm not sure if `value` is set if there is no `var_name`, or add `None` defaults to each `get`.

Reply:
`get` defaults to None if a default is not specified.